presto-main/src/test/java/com/facebook/presto/sql/analyzer/TestAnalyzer.java

```diff
@@ -469,6 +469,14 @@ public void testGroupByWithQualifiedName3()
     }
 
     @Test
+    public void testGroupByWithDistinct()
+            throws Exception
+    {
+        // TODO: verify output
+        analyze("SELECT distinct(a, b), c FROM t1 GROUP BY t1.a, t1.b, t1.c");
```

Comment:
This formatting is confusing because it makes `distinct()` look like a function call. It would be better to write it like this:
```sql
SELECT DISTINCT (a, b), c FROM t1 GROUP BY t1.a, t1.b, t1.c
```
Or if this works:
```sql
SELECT DISTINCT a, (b, c) FROM t1 GROUP BY t1.a, t1.b, t1.c
```
Also, do the group by expressions need the `t1.` qualifier to reproduce the issue?